src/app/legacy/containers/PageHandlers/withVariant/index.jsx

```diff
@@ -9,6 +9,7 @@ const WithVariant = Component => {
 
     const { path, params } = pathOr({}, ['match'], props);
 
+    // Still need to do something about this
```

Comment:
I don't think its fully confirmed, but I think we'll be dropping the auto-redirect cookie stuff. @andrewscfc may be able to chip in here.